packages/local_auth/local_auth_darwin/darwin/local_auth_darwin.podspec

```diff
@@ -2,7 +2,7 @@
 # To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
```

Comment:
Actually this needs the `PrivacyInfo.xcprivacy` pulled in and `resource_bundles` path updated:
```
 -> local_auth_darwin (0.0.1)
    - NOTE  | url: The URL (https://pub.dev/packages/local_auth_darwin) is not reachable.
    - ERROR | [iOS] file patterns: The `resource_bundles` pattern for `local_auth_ios_privacy` did not match any file.
```

See https://github.com/flutter/packages/pull/5846/files#diff-3224e4fc5e9a47bae7d7c5dcc243e869174b97b0b4e8572a6e58fde8c0b3f5d5